university-of-presov.csl

```diff
@@ -0,0 +1,359 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" delimiter-precedes-last="never" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>University of Presov</title>
+    <title-short>UoP</title-short>
+    <id>http://www.zotero.org/styles/university-of-presov</id>
+    <link href="http://www.zotero.org/styles/university-of-presov" rel="self"/>
+    <link href="http://www.zotero.org/styles/university-of-presov" rel="template"/>
+    <author>
+      <name>Peter L Jones</name>
```

Comment:
Is this you?
If not, please give credit via the template link as mentioned above and put yourself as the author.